README.md

```diff
@@ -17,6 +19,10 @@ The MSAL preview library for iOS and macOS gives your app the ability to begin u
 * How to call the Microsoft Graph API
 * How to sign a user out of your application
 
+## Scenario
+
+This app is a multi-tenant app meaning it can be used by any Azure AD tenant or Microsoft Account.  It demonstrates how a developer can build apps to connect with enterprise users and access their Azure + O365 data via the Microsoft Graph.  During the auth flow, end users will be required to sign in and consent to the permissions of the application, and in some cases may require an admin to consent to the app.  The majority of the logic in this sample shows how to auth an end user and make a basic call to the Microsoft Graph.
```

Comment:
You might want to add a picture of the topology (I can do it for you if you wish), and a screen copy of the UI?

Reply:
I believe that - instead of describing the scenario too much here, we should just point people to the https://docs.microsoft.com/en-us/azure/active-directory/develop/guidedsetups/active-directory-ios instead